client/src/components/web-extension/CreateWallet/ValidateKeyphrasePage.jsx

```diff
@@ -6,21 +6,28 @@ import every from 'lodash-es/every';
 import { generateCWHeader, onGenerateWordcheck } from '@cd/actions/createWalletActions.utils';
 import { setNextStep, updateAnswerSheet, createAnswerSheet } from '@cd/actions/createWalletActions';
 import { selectCreateWalletState } from '@cd/selectors/createWallet';
+import { CONSTANTS } from '@cd/shared/constants';
 import WordsGroup from './WordsGroup';
 import './ValidateKeyphrase.scss';
 
 const ValidateKeyphrasePage = () => {
 	const [, setHeader] = useOutletContext();
 	const dispatch = useDispatch();
-	const { currentStep, answerSheet, keyPhraseAsMap, totalKeywords, totalWordCheck } =
-		useSelector(selectCreateWalletState);
+	const { currentStep, answerSheet, keyPhraseAsMap } = useSelector(selectCreateWalletState);
 	const [wordsTemplate, setTemplate] = useState(undefined);
 	const onUpdateAnswerSheet = useCallback(
 		(groupIndex, value) => dispatch(updateAnswerSheet(groupIndex, value)),
 		[dispatch],
 	);
 	const onCreateAnswerSheet = useCallback((checklist) => dispatch(createAnswerSheet(checklist)), [dispatch]);
+
+	const totalWordCheck = keyPhraseAsMap.size / 3;
```

Comment:
Not sure why we removed `totalWordCheck` from redux. That's a good place for initial value, instead do some math like this

Reply:
IMO`totalWordCheck` state in redux is redundant, we still need to do math and update this state each time number of recovery word changes